SETUP.md

```diff
@@ -49,6 +51,63 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
 
 ## OS-specific prerequisites
 
+### OS X Catalina
+([Older Versions](#os-x-mojave--mavericks--yosemite--el-capitan--sierra))
+
+1. Install Homebrew: `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+1. Install Redis: `brew install redis`
+1. Run `brew install https://raw.githubusercontent.com/turforlag/homebrew-cervezas/master/pdftk.rb enscript gs mysql@5.7 nvm imagemagick rbenv ruby-build coreutils sqlite parallel`
+    <details>
+        <summary>Troubleshoot: old version of `&lt;package&gt;`</summary>
+
+        If it complains about an old version of `<package>`, run `brew unlink <package>` and run `brew install <package>` again
+    </details>
+1. Configure your shell
+    1. Add the following to `~/.bash_profile` or your desired shell configuration file:
+        ```
+        # Start rbenv
```

Comment:
I kind of liked having the shell changes inline with each item but don't feel strongly about it either way.